virttest/env_process.py

```diff
@@ -1147,6 +1148,11 @@ def preprocess(test, params, env):
             session.close()
             test.cancel("Failed to map hostname and ipaddress of target host")
         session.close()
+        if params.get("setup_ssh") == "yes":
```

Comment:
perhaps move those logic change to provider under tp-libvirt considering it is specific libvirt migration feature

Reply:
No, ssh passwordless connection is required by preprocess and postprocess. Moving to tp-libvirt does not work in this case.